<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Програма для перетворень</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Лаба №1</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>261</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
        qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 0px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
        font-size: 18px;
}</string>
      </property>
      <property name="text">
       <string>Точки</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border: 1px solid #FF17365D;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="Point_0_0">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Point_0_1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Point_1_0">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Point_1_1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Point_2_0">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Point_2_1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Point_3_0">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Point_3_1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>271</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
        qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 0px;
	border-top-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
        font-size: 18px;
}</string>
      </property>
      <property name="text">
       <string>Фігура</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>271</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border: 1px solid #FF17365D;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-top-right-radius: 0px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QRadioButton" name="radioButton_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton{	
	font: 14pt &quot;MS Shell Dlg 2&quot;;
}</string>
       </property>
       <property name="text">
        <string>Точки</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton{	
	font: 14pt &quot;MS Shell Dlg 2&quot;;

}</string>
       </property>
       <property name="text">
        <string>Лінія</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton{	
	font: 14pt &quot;MS Shell Dlg 2&quot;;

}</string>
       </property>
       <property name="text">
        <string>Трикутник</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>261</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
        qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;

	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
        font-size: 18px;
}</string>
      </property>
      <property name="text">
       <string>Матриця</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>261</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border: 1px solid #FF17365D;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 0px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="Matrix_0_1">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Matrix_0_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Matrix_1_1">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Matrix_1_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Matrix_2_0">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>150</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Matrix_2_1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>150</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	qproperty-alignment: AlignCenter;
	border: 2px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>271</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
        qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
        font-size: 18px;
}</string>
      </property>
      <property name="text">
       <string>Операції</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>271</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border: 1px solid #FF17365D;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 15px;
	border-top-right-radius: 0px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="convert">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-alignment: AlignCenter;
	border: 1px solid #FF17365D;
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	background-color: #FF17365D;
	padding: 5px 0px;
	color: rgb(255, 255, 255);
	max-height: 25px;
    font-size: 18px;
}</string>
       </property>
       <property name="text">
        <string>Перетворити</string>
       </property>
      </widget>
     </widget>
     <zorder>frame_5</zorder>
     <zorder>frame_3</zorder>
     <zorder>label_3</zorder>
     <zorder>frame_2</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>frame_4</zorder>
     <zorder>label_15</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Лаба №2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Лаба №3</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Про програму</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
